snippets/csharp/getting_started/ClassLibraryProjects/ShowCase/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using UtilityLibraries;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        int row = 0; 
+
+        do
+        {
+            if (row == 0 || row >= 25)
+               ResetConsole();
+
+            string input = Console.ReadLine();
+            if (String.IsNullOrEmpty(input)) break;
+            Console.WriteLine($"Input: {input} {"Begins with uppercase? ",30}: " +
+                              $"{(input.StartsWithUpper() ? "Yes" : "No")}\n");
```

Comment:
I feel it's less noisy that way.

```suggestion
                              input.StartsWithUpper() ? "Yes\n" : "No\n";
```